admin/app/components/solidus_admin/base_component.rb

```diff
@@ -4,13 +4,6 @@ module SolidusAdmin
   # BaseComponent is the base class for all components in Solidus Admin.
   class BaseComponent < ViewComponent::Base
     include ViewComponent::InlineTemplate
-
-    def container
-      SolidusAdmin::Container
-    end
-
-    def component(name)
-      container.resolve("#{name}_component")
-    end
+    include SolidusAdmin::ContainerHelper
```

Comment:
What about `SolidusAdmin::ComponentsHelper`? I would like to avoid adding a technical implementation detail to our public API. I prefer naming that explains its usecase rather than its implementation.

Reply:
See https://github.com/solidusio/solidus/pull/5099#discussion_r1207108938